toplevel/byte/topmain.ml

```diff
@@ -203,8 +203,15 @@ end)
 let main () =
   let ppf = Format.err_formatter in
   let program = "ocaml" in
+  let deprecated_script_alert =
+    if Array.length !argv < 2 then
+      None
+    else
+      Warnings.deprecated_script_alert !argv.(1)
+  in
```

Comment:
Should we also add this alert to `ocamlnat` in order to reduce code divergence between the two toplevels?